plugins/module_utils/constants.py

```diff
@@ -0,0 +1,6 @@
+# coding: utf-8 -*-
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
```

Comment:
Does it make sense to move https://github.com/flightctl/flightctl-ansible/pull/4/files#diff-5610ae59ef2d2d3dd2b4c2c6f6714b3ca717790cf804603f03fcac3a34d3d162L84 here?

Reply:
I think its ok for the api module to own the endpoint definitions as it seems like the proper "owner" for that info and right now I don't think anything else requires that data.  

However i do think that given I've added definitions for two of the kind/resource types in the constants file maybe all of them should be defined there and possibly something like an enum could be used in the future to better tie up those values (and those could then be referenced in the API_ENDPOINTS definition).